suites/pacific/rgw/tier_1_object.yaml

```diff
@@ -1,7 +1,8 @@
-"""
-Note: If the following tests are being called and executed manually or via pipeline.
-      Please use conf/pacific/rgw/tier_0_rgw.yaml
-"""
+# Tier-1: Downstream RGW test suite focusing on verifying the core features.
+
+# This test suite is auto triggered in the stage 1 of QE pipeline. The suite requires
+# a single cluster deployment having only one RGW endpoint.
```

Comment:
along with this note, I still think we should add this `conf/pacific/rgw/tier_0_rgw.yaml` for reference

Reply:
done